packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -377,10 +377,23 @@ export default class TradeStore extends BaseStore {
     onPurchase(proposal_id, price, type) {
         if (!this.is_purchase_enabled) return;
         performance.mark('purchase-started');
+
         if (proposal_id) {
             this.is_purchase_enabled = false;
             const is_tick_contract = this.duration_unit === 't';
-            processPurchase(proposal_id, price).then(action((response) => {
+            const passthrough = {
```

Comment:
can you add a comment that this is for logging additional info to the API in case of errors occurring? since it's not obvious otherwise why a passthrough is here when it's unused in the FE codebase

Reply:
Sure. Will do.